app/src/main/java/grinnell/appdev/edu/lyles/MainActivity.java

```diff
@@ -20,38 +23,57 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
+        //Set toolbar as the action bar
         Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
         setSupportActionBar(toolbar);
 
         final DrawerLayout drawer = (DrawerLayout) findViewById(R.id.drawer);
 
+        //Enable navigation drawer buttons
         NavigationView nav = (NavigationView) findViewById(R.id.nav_view);
         nav.setNavigationItemSelectedListener(new NavigationView.OnNavigationItemSelectedListener() {
             @Override
             public boolean onNavigationItemSelected(MenuItem item) {
+                //Set "checked" highlight
                 if(item.isChecked()) item.setChecked(false);
                 else item.setChecked(true);
 
+                //Close the drawer when an item is selected
                 drawer.closeDrawers();
 
                 Fragment newFrag;
```

Comment:
@PsychicNoodles How about we change this to something more descriptive? (eg. newFragment, fragment, initialFragment)